src/legacy/core_plugins/console/public/quarantined/src/autocomplete/components/template_autocomplete_component.js

```diff
@@ -21,7 +21,7 @@ import { ListComponent } from './list_component';
 
 export class TemplateAutocompleteComponent extends ListComponent {
   constructor(name, parent) {
-    super(name, mappings.getTemplates, parent);
+    super(name, mappings.getTemplates, parent, true, true);
```

Comment:
Are these args related to this statement: `Updates completion to kick in even if index in URL does not match an existing endpoint`?

Also, note that the url contains the template name, not the index (not sure if you just had a typo there or not).

Reply:
> Are these args related to this statement:...

Yes, :(. Sorry that it's a bit unclear but I would very much like to get to properly documenting how the system works. The object being extended is `ListComponent` which is where you would be able to better see what those arguments do.

> Also, note...

Ah fair enough! That was my mistake, will update PR description.